packages/ui5-proxy-middleware/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    transform: {
+        '^.+\\.ts$': 'ts-jest'
+    },
+    collectCoverage: true,
+    coverageReporters: ['text', 'html'],
```

Comment:
Support for Sonar coverage reporting needs to be added, otherwise Sonar is going to report 0% coverage and the PR build will not pass. Please look at changes to `jest.config.js` and `sonar-project.properties` in this commit: https://github.com/SAP/open-ux-tools/commit/f6c7309a13a1c62cffac75271be16984d7cbdf63

@kranthie-sap, we may need to add a quick note to the README or a dev guide how to do this (unless it's already done!).

Reply:
Thanks, done with https://github.com/SAP/open-ux-tools/pull/249/commits/1cb54b0b6f3d2c3e9071b9d5991ce9cffe08b2f4